bullet/client.py

```diff
@@ -224,6 +224,127 @@ def launch(self, default = None):
             if ret is not None:
                 return ret
 
+
+@keyhandler.init
+class BulletDict:
```

Comment:
This is a lot of repeated code for what seems like it should just be a child class that overrides accept().

Reply:
ah okay that makes sense ill do that now thank you.

Reply:
what are your thoughts on it now? any improvements you would recommend me trying to do?